diff --git a/node_modules/@docusaurus/plugin-content-docs/lib/docs.js b/node_modules/@docusaurus/plugin-content-docs/lib/docs.js
index f0756ed..4162272 100644
--- a/node_modules/@docusaurus/plugin-content-docs/lib/docs.js
+++ b/node_modules/@docusaurus/plugin-content-docs/lib/docs.js
@@ -115,6 +115,7 @@ function processDocMetadata({ docFile, versionMetadata, context, options, }) {
         lastUpdatedBy: lastUpdate.lastUpdatedBy,
         lastUpdatedAt: lastUpdate.lastUpdatedAt,
         sidebar_label,
+        files: frontMatter.files ? frontMatter.files : ''
     };
 }
 exports.processDocMetadata = processDocMetadata;
