diff --git a/node_modules/@docusaurus/plugin-content-docs/lib/docs.js b/node_modules/@docusaurus/plugin-content-docs/lib/docs.js
index 981e250..235f41f 100644
--- a/node_modules/@docusaurus/plugin-content-docs/lib/docs.js
+++ b/node_modules/@docusaurus/plugin-content-docs/lib/docs.js
@@ -114,6 +114,7 @@ function processDocMetadata({ docFile, versionMetadata, context, options, }) {
         lastUpdatedBy: lastUpdate.lastUpdatedBy,
         lastUpdatedAt: lastUpdate.lastUpdatedAt,
         sidebar_label,
+        files: frontMatter.files ? frontMatter.files : ''
     };
     return metadata;
 }
